Atala PRISM/2.10.md

```diff
@@ -0,0 +1,118 @@
+# Atala PRISM v2.10
+
+March, 2024 release of Atala PRISM.
+
+## Components
+
+### Platform
+
+* [Open Enterprise Agent v1.28.0](https://github.com/hyperledger-labs/open-enterprise-agent/releases/tag/prism-agent-v1.28.0)
+* [PRISM Node v2.2.1](https://github.com/input-output-hk/atala-prism/releases/tag/v2.2.1)
+* [PRISM Mediator v0.14.1](https://github.com/input-output-hk/atala-prism-mediator/releases/tag/prism-mediator-v0.14.1)
+
+### SDKs
+
+* [Apollo Crypto Library v1.2.10](https://github.com/input-output-hk/atala-prism-apollo/releases/tag/v1.2.10)
+* [Wallet SDK Swift v5.0.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift/releases/tag/5.0.0)
+* [Wallet SDK TypeScript v5.0.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts/releases/tag/v5.0.0)
+* [Wallet SDK KMM v3.0.0](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm/releases/tag/v3.0.0)
+
+### Documentation
+
+* [Documentation Portal v1.81.0](https://github.com/input-output-hk/atala-prism-docs/releases/tag/v1.81.0)
+
+## Changelog
```

Comment:
We should have also the changes for Mediator in the release note.

Reply:
Those are the main points:

Support protocol discover-features 2.0
Support for websockets
Support for Live Mode in the Pickup protocol
Return 202 HTTP code (for DID Comm)
Re-enable the logging based on X-Request-Id header
Minor fixes
Use Java 11 runtime
Update libs